### ISSUE TITLE
GameGuru MAX - Bug/Request - Change Texture Needs Emissive

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4060]
I consider this a bug just because it's so necessary, but Change Texture needs a way to add an emissive texture or at least allow for emissive to work with it somehow. Currently, it doesn't seem possible.

**Comment by davetgc** [comment]
This is a missing feature more than a bug, but it an urgent addition, requested by a number of DLC artists.

**Comment by ZakJudges** [comment]
@MonkeyFrogStudio New EXP build uploaded to Steam with a fix for this:

A scan for PBR textures will now take place. So, if the image that you select in the behavior panel is a _color texture, then any textures with _normal, _surface, _emissive will also be applied to the object that the script is attached to.

In this example, I copied one of your PBR texture sets from the Offworld Station booster into the image bank. Added the Change Texture behavior to an object, then selected the _color texture as IMAGEFILE1 in the behavior panel, like so:
![Screenshot 2023-01-26 104142](https://user-images.githubusercontent.com/32545391/214822643-bfc685f0-9df2-4c1a-9131-64f76a52c74e.png)

This now results in the full PBR set being applied to the object:
![Screenshot 2023-01-26 104218](https://user-images.githubusercontent.com/32545391/214822932-cc36120c-f8fc-4078-a512-a5a4688d4656.png)

Also, added emissive strength settings to the behavior, for finer control :)
